systemtest/rum_test.go

```diff
@@ -65,7 +66,21 @@ func TestRUMXForwardedFor(t *testing.T) {
 		"@timestamp", "timestamp.us",
 		// RUM events have the source port recorded, and in the tests it will be dynamic
 		"source.port",
+		// Do not assert the exact contents of the location field since they may change
+		// slightly depending on the IP lookup.
+		"client.geo.location",
 	)
+
+	var location = make(map[string]interface{})
+	var ip string
+	for _, doc := range result.Hits.Hits {
+		client := doc.Source["client"].(map[string]interface{})
+		ip = client["ip"].(string)
+		location = client["geo"].(map[string]interface{})["location"].(map[string]interface{})
+	}
+	assert.Contains(t, location, "lat")
+	assert.Contains(t, location, "lon")
+	assert.Equal(t, ipAddress, ip)
```

Comment:
We know that geoIP enrichment has been effective due to the `city_name` etc. being set. We also see the IP address in the approbal. Are these new assertions (lines 74+) testing anything additional?

Reply:
Probably not adding anything and just overcasting this test case, I'll remove them since it's clear that we're already asserting the output with the document check.